src/MailerMessage.php

```diff
@@ -335,6 +337,23 @@ public function setCustomCampaign($campaign)
         return $this;
     }
  
+        /**
+     * @inheritdoc
+     */
+    public function getCustomID()
```

Comment:
+ Please use getCustomId
+ Its not an inherit documented function and therefore needs php doc
+ please provide the phpdoc @see property with a link to the mailjet api which documents what CustomID is.
+ add since tag